node/clients/masterdata/onboarding.ts

```diff
@@ -97,6 +97,7 @@ export class Onboarding extends MasterData {
 
       return response[0] || null
     } catch (err) {
+      // eslint-disable-next-line no-console
```

Comment:
Using console.error instead of console.log should allow you to keep this eslint rule enabled